packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -72,25 +117,96 @@ const DialoguesList = ({ classes }: { classes: ClassesType }) => {
     fragmentName: "PostsListWithVotes",
     resolverName: "MyDialogues",
     limit: 3,
-  }); 
+  });
+
+  const {
+    matchedUsersQueryResult: { data: matchedUsersResult },
+    userDialogueChecksResult: { results: userDialogueChecks = [] },
+  } = useDialogueMatchmaking({ getMatchedUsers: true, getOptedInUsers: false, getUserDialogueChecks: true });
 
-  const dialoguesTooltip = <div>
+  const matchedUsers: UsersOptedInToDialogueFacilitation[] | undefined = matchedUsersResult?.GetDialogueMatchedUsers;
+
+  const dialoguesTooltip = (<div>
     <p>Dialogues between a small group of users. Click to see more.</p>
-  </div>
+  </div>);
 
   const renderMyDialogues = !!currentUser && myDialogues?.length 
 
-  const myDialoguesTooltip = <div>
-      <div>These are the dialoges you are involved in (both drafts and published)</div>
-    </div>
+  const myDialoguesTooltip = (<div>
+    <div>These are the dialogues you are involved in (both drafts and published)</div>
+  </div>);
+
+  const matchmakingTooltip = (<div>
+    <p>Users you've already matched with for dialogues.  Click here to go to the dialogue matchmaking page.</p>
+  </div>);
+
+  const rowPropsList = currentUser && getRowProps<false>({
+    currentUser,
+    isUpvotedUser: false,
+    showAgreement: false,
+    showBio: false,
+    showFrequentCommentedTopics: false,
+    showKarma: false,
+    showPostsYouveRead: false,
+    userDialogueChecks,
+    users: matchedUsers ?? []
+  });
 
   return <AnalyticsContext pageSubSectionContext="dialoguesList">
     <SingleColumnSection>
       <SectionTitle href="/dialogues"
         title={<LWTooltip placement="top-start" title={dialoguesTooltip}>
           Dialogues
         </LWTooltip>}
-      />
+      >
+        <SectionButton>
+          <MuiPeopleIcon />
+          <Link to="/dialogueMatching">Find Dialogue Partners</Link>
+        </SectionButton>
+      </SectionTitle>
+
+      <AnalyticsContext pageSubSectionContext="frontpageDialogueMatchmaking">
```

Comment:
Doesn't have to be addressed for this PR, but really these additions to this component should be in their own component. More modular. Means if you ever wanted to remove them, can do, ever want reuse elsewhere, can do.

Reply:
Agreed! Will do this in future PR!